addon/components/sortable-group.js

```diff
@@ -674,5 +651,38 @@ export default Component.extend({
    */
   activateKeyDown(){
     this.set('isKeyDownEnabled', true);
-  }
+  },
+
+  /**
+   * Defining getters and setters to support native getter and setters until we decide to drop support for ember versions below 3.10
+   */
+  _setGetterSetters() {
```

Comment:
will it work defined in object itself like `get itemPosition() { ... }`?

Reply:
I think so, the ember try passed when it used to fail.
@pzuraq?

Reply:
@lifeart oh wait sorry I misread, directly using getter and setter like that is backward incompatibility. I originally tried that

Reply:
You could maybe convert the component to an es class and then use getters. Not sure what the support matrix is though

Reply:
@knownasilya At that point, I think I would just be doing the Octane migration then 😆 
I think we need to decide which point in the future, we should just outright deprecate all incompatible Ember versions.

Reply:
According to the blog post you can get 3.4+ with a polyfill. Glimmer will be 3.14 which will leave many behind.
https://blog.emberjs.com/2019/01/26/emberjs-native-class-update-2019-edition.html

Reply:
I was looking at the polyfill, but to my understanding I don't think it'll solve this particular issue right? Just by using the polyfill doesn't mean that I can now use native getter and setter in 3.4+? It's more of an effort to ease the migration of the entire class to native class, so it would probably not do anything regarding deprecating `volatile` and etc?

Reply:
Or maybe you were just talking with regards to the future plans to move to native classes and not this particular problem 😄